formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/ProtoBuf.kt

```diff
@@ -404,66 +458,71 @@ public class ProtoBuf(
         fun nextInt(format: ProtoNumberType): Int {
             val wireType = if (format == ProtoNumberType.FIXED) i32 else VARINT
             assertWireType(wireType)
-            val ans = decode32(format)
+            val result = decode32(format)
             readTag()
-            return ans
+            return result
         }
 
         fun nextLong(format: ProtoNumberType): Long {
             val wireType = if (format == ProtoNumberType.FIXED) i64 else VARINT
             assertWireType(wireType)
-            val ans = decode64(format)
+            val result = decode64(format)
             readTag()
-            return ans
+            return result
         }
 
         fun nextFloat(): Float {
             assertWireType(i32)
-            val ans = readIntLittleEndian()
+            val result = Float.fromBits(readIntLittleEndian())
             readTag()
-            return Float.fromBits(ans)
+            return result
         }
 
         private fun readIntLittleEndian(): Int {
+            // TODO this could be optimized by extracting method to the IS
```

Comment:
Is this functionality going to be in kotlinx-io?

Reply:
This one is likely to